docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,828 @@
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>0</th>\n",
+       "      <th>1</th>\n",
+       "      <th>2</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>-4.822819e+06</td>\n",
+       "      <td>-4.778850e+06</td>\n",
+       "      <td>-4.842920e+06</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2.883705e+05</td>\n",
+       "      <td>2.554854e+05</td>\n",
+       "      <td>3.032475e+05</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>4.827303e+05</td>\n",
+       "      <td>4.255387e+05</td>\n",
+       "      <td>5.069975e+05</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>-5.005799e+05</td>\n",
+       "      <td>-4.454748e+05</td>\n",
+       "      <td>-5.240727e+05</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>-8.573380e+05</td>\n",
+       "      <td>-8.781561e+05</td>\n",
+       "      <td>-8.504175e+05</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>2.711764e+06</td>\n",
+       "      <td>2.721883e+06</td>\n",
+       "      <td>2.709151e+06</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>5.622326e+05</td>\n",
+       "      <td>6.523093e+05</td>\n",
+       "      <td>5.230883e+05</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "              0             1             2\n",
+       "0 -4.822819e+06 -4.778850e+06 -4.842920e+06\n",
+       "1  2.883705e+05  2.554854e+05  3.032475e+05\n",
+       "2  4.827303e+05  4.255387e+05  5.069975e+05\n",
+       "3 -5.005799e+05 -4.454748e+05 -5.240727e+05\n",
+       "4 -8.573380e+05 -8.781561e+05 -8.504175e+05\n",
+       "5  2.711764e+06  2.721883e+06  2.709151e+06\n",
+       "6  5.622326e+05  6.523093e+05  5.230883e+05"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "print(\"------node voltage result------\")\n",
+    "display(pd.DataFrame(asym_result[\"sym_voltage_sensor\"][\"u_residual\"]))\n",
+    "\n",
+    "print(\"------sym_load result------\")\n",
+    "display(pd.DataFrame(asym_result[\"sym_power_sensor\"][\"p_residual\"]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9330460f-186c-4e44-8a8a-012f99607530",
+   "metadata": {},
+   "source": [
+    "For the observability and batch calculation of state estimation, we refer to `State Estimation Example`."
```

Comment:
here and maybe in other locations: you can cross-reference notebooks by using e.g. `[Validation Examples](./Validation%20Examples.ipynb)`

```suggestion
    "For the observability and batch calculation of state estimation, we refer to the [State Estimation Example](./State%20Estimation%20Example.ipynb)."
```